src/openms/source/CHEMISTRY/EnzymesDB.cpp

```diff
@@ -53,12 +53,11 @@ namespace OpenMS
   EnzymesDB::EnzymesDB()
   {
     readEnzymesFromFile_("CHEMISTRY/Enzymes.xml");
```

Comment:
Strange, because IDMerger crashes for me with the error given above. All the tests for it ("ctest -R TOPP_IDMerger") fail, too. Other ID... tools are also affected.